CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing
+
+## Overview
+
+This documents explains the processes and practices recommended for contributing enhancements to
+this operator.
+
+- Generally, before developing enhancements to this charm, you should consider [opening an issue
+  ](https://github.com/canonical/nats-operator/issues) explaining your use case.
```

Comment:
this should refer to https://github.com/canonical/lxc-integrator-operator/issues